### PR TITLE
Kwisi tampa

### LINKED ISSUE #22: kwisi - Tampa, FL
## Tampa, FL

- [x] Create a branch
- [x] Add File
- [x] Create a commit
- [ ] Open a PR
- [ ] Have a discussion about PR
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by Kwisi**
Closes #22

### CODE REVIEW

**Comment by Kwisi** [comment]
@beardofedu I was told to flood you with notifications

File: places/kwisi-Tampa.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Tampa, FL
+
+### Fun Facts
+Pirates, everywhere
```
Comment (wordsoup) [inline places/kwisi-Tampa.md]: 😆

**Comment by beardofedu** [comment]
@Kwisi definitely jealous about the Bern's for dinner!

File: places/kwisi-Tampa.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Tampa, FL
+
+### Fun Facts
+- Pirates, everywhere.  We have a yearly event called 'Gasparilla,' which is basically pirate Mardi Gras.
+
+### Places to Eat
+- Bern's Steakhouse (I'm going there tonight, everyone should be jealous)
+
+### Places to Go
+- Busch Gardens - just got a new roller coaster this year
```
Comment (readiu) [inline places/kwisi-Tampa.md]: Top Golf!!!

Comment (Kwisi) [inline places/kwisi-Tampa.md]: My co-workers love it, but golf isn't my thing.

**Pull request merged**